### PR TITLE
ORCA-157| updated the file with new SQS variable names and output des…

### PR DESCRIPTION
**Pull request by rizbi-hassan**
## Summary of Changes
Updated ORCA github documentation to add new variable names, output and descriptions that were added while creating SQS queues.

Addresses [ORCA-158: Update ORCA github documentation to add new variable names and descriptions that were added while creating SQS queues.](https://bugs.earthdata.nasa.gov/browse/ORCA-158)

## Changes
cumulus-orca/website/docs/developer/deployment-guide/deployment-with-cumulus.md
* added new SQS variable names, descriptions, default values and output.
* formatted indentation to make the variables look good.

## Type of change

- [x] New feature (non-breaking change which adds functionality)

## How Has This Been Tested?

The variables in SQS were deployed using "terraform apply" and no errors were reported during the process.

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the necessary documentation (remove those that do not apply)
    - [ ] CHANGELOG.md
    - [ ] Testing documentation
    - [ ] Development documentation
    - [ ] User documentation
    - [ ] (Additional here)
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
    - [ ] Unit tests
    - [ ] Integration tests
    - [ ] Manual tests (outlined above)
- [ ] New and existing unit tests pass locally with my changes
    - [ ] Automated tests passing (if not fork)
    - [ ] Manual testing/integration testing passes (if forked)
- [ ] Any dependent changes have been merged and published in downstream modules
- [ ] My code has passed security scanning
    - [ ] Snyk
    - [ ] git-secrets

### CODE REVIEW

**Review by bhazuka — COMMENTED** [review]
Looks good.

File: website/docs/developer/deployment-guide/deployment-with-cumulus.md [hunk 1]
```diff
@@ -80,16 +80,20 @@ module "orca" {
...
+  # orca_recovery_complete_filter_prefix         = ""
+  # orca_recovery_expiration_days                = 5
+  # orca_recovery_lambda_memory_size             = 128
+  # orca_recovery_lambda_timeout                 = 300
+  # orca_recovery_retry_limit                    = 3
+  # orca_recovery_retry_interval                 = 1
+  # sqs_delay_time                               = 0
+  # sqs_maximum_message_size                     = 262144
+  # staged_recovery_queue_message_retention_time = 432000
+  # status_update_queue_message_retention_time   = 777600
```
Comment (AndrewDorn) [inline website/docs/developer/deployment-guide/deployment-with-cumulus.md:96]: I would personally vote we add units to the 3 names.

Comment (rizbi-hassan) [reply website/docs/developer/deployment-guide/deployment-with-cumulus.md:96]: yes I agree. It looks a bit confusing without knowing the variable description.

Comment (rizbi-hassan) [reply website/docs/developer/deployment-guide/deployment-with-cumulus.md:96]: @bhazuka should I make this change to the names? I know we merged the changes for https://bugs.earthdata.nasa.gov/browse/ORCA-157 last week and closed the card.

Comment (bhazuka) [reply website/docs/developer/deployment-guide/deployment-with-cumulus.md:96]: Lets put a card in to update the names that way you can handle it across both. That way we can discuss prior to see if there are other variable names and or adjustments to make.

Comment (rizbi-hassan) [reply website/docs/developer/deployment-guide/deployment-with-cumulus.md:96]: created [ORCA-158](https://bugs.earthdata.nasa.gov/browse/ORCA-158) to address this in the future.

**Pull request merged**